cluster/ci/config/prow/config.yaml

```diff
@@ -379,6 +379,32 @@ presubmits:
     rerun_command: "/test service-catalog-openshift"
     trigger: "((?m)^/test service-catalog-openshift,?(\\s+|$))"
 
+  - name: test_pull_request_origin_extended_conformance_azure
+    agent: jenkins
+    labels:
+      master: ci.openshift.redhat.com
+    always_run: true
+    # TODO: Remove once the job is stable
+    skip_report: true
+    branches:
+    - master
+    context: ci/openshift-jenkins/azure
+    rerun_command: "/test azure"
+    trigger: "((?m)^/test( all| azure),?(\\s+|$))"
+
+  - name: test_pull_request_origin_extended_conformance_azure_39
+    agent: jenkins
+    labels:
+      master: ci.openshift.redhat.com
+    always_run: true
+    # TODO: Remove once the job is stable
+    skip_report: true
+    branches:
+    - release-3.9
+    context: ci/openshift-jenkins/azure
+    rerun_command: "/test azure"
```

Comment:
@pweil- you had raised discussion about the experience around launching these jobs, does this lgty?

Reply:
:+1: